### PR TITLE
Add a test for uploads/image serving

### PR DESCRIPTION
**Pull request by rbuysse**

### CODE REVIEW

**Comment by rbuysse** [comment]
To test: 
1) Fetch PR branch
2) `just test`

**Comment by shmup** [comment]
Tests run successfully on my system, but I had to make this change:

```diff
-  docker compose -f tests/docker-compose.yaml build
+  docker-compose -f tests/docker-compose.yaml build
```

File: justfile [hunk 1]
```diff
@@ -21,3 +21,15 @@ fix-fmt:
...
   go run main.go
 
+
+test:
+  #!/usr/bin/env sh
+  set -e
+
+  trap "docker-compose -f tests/docker-compose.yaml down" EXIT
+
+  docker compose -f tests/docker-compose.yaml build
```
Comment (shmup) [inline justfile]:
```diff
- docker compose ...
+ docker-compose ...
```

Comment (shmup) [reply justfile]: otherwise tests run fine though!

**Review by shmup — APPROVED** [review]

**Pull request merged**